test/remote/gateways/remote_adyen_test.rb

```diff
@@ -1090,12 +1093,12 @@ def test_purchase_using_stored_credential_unscheduled_mit
     assert auth = @gateway.authorize(@amount, @credit_card, initial_options)
     assert_success auth
     assert_equal 'UnscheduledCardOnFile', auth.params['additionalData']['recurringProcessingModel']
+
     assert capture = @gateway.capture(@amount, auth.authorization)
     assert_success capture
     assert_equal '[capture-received]', capture.message
-    assert network_transaction_id = 'none'
 
-    used_options = stored_credential_options(:unscheduled, :cardholder, id: network_transaction_id)
+    used_options = stored_credential_options(:unscheduled, :cardholder, id: auth.network_transaction_id)
```

Comment:
I just noticed all of the remote tests are using `cardholder` for the followup transaction, which means the NTID would not actually be sent for any of these. I will add a few for merchant-initiated followups.